python/nav/web/api/v1/views.py

```diff
@@ -1423,3 +1410,133 @@ def post(self, request):
             'refresh_token': refresh_token,
         }
         return Response(response_data)
+
+
+class MaintenanceTaskViewSet(NAVAPIMixin, APIView):
+    """
+    A ViewSet for posting MaintenanceTasks.
+
+    Responds with a JSON dict representation of the created maintenance task or an error
+    message.
+
+    Example POST request:
+        `/api/1/maintenance/` with body
+        `{
+            "start_time": "2025-09-29T11:11:11",
+            "end_time": "2025-09-29T13:11:11",
+            "no_end_time": false,
+            "description": "Changing out old equipment in serverroom",
+            "location": [1, 2],
+            "room": ["myroom", "secondroom"],
+            "netbox": [1, 2],
+            "service": [1, 2],
+            "netboxgroup": [1, 2]
+         }`
+
+    Example POST response:
+        `{
+            "id": 59,
+            "maintenance_components": "[<Netbox: buick.lab.uninett.no>,
+                <Netbox: oldsmobile.lab.uninett.no>]",
+            "start_time": "2025-10-01T09:17:00",
+            "end_time": "9999-12-31T23:59:59.999999",
+            "description": "Changing out old equipment in serverroom",
+            "author": "admin",
+            "state": "scheduled"
+        }`
+    """
```

Comment:
The POST body and the response body should preferably be symmetric when dealing with model API calls.  These are definitely not symmetric (e.g. I can't take the response body, remove the `id` attribute and post it again as new record).